ios/tos/tos/Views/PopupMenuView.swift

```diff
@@ -46,7 +64,9 @@ struct PopupMenuView: View {
         }
         .padding()
         .background(RoundedRectangle(cornerRadius: 16)
-            .stroke(Color.black, lineWidth: 2)
-            .background(Color.white))
+                        .stroke(Color.black, lineWidth: 2)
```

Comment:
formatting

https://github.com/nicklockwood/SwiftFormat I have this set up for xcode and it works well